portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,51 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<String> quotes = createQuotesArray();
+    String quotesJson = convertToJson(quotes);
+
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println(quotesJson);
+  } 
+
+  /**
+   * This function creates an ArrayList containing a few hard-coded quotes. 
+   */ 
+  private ArrayList createQuotesArray() {
```

Comment:
This method can return List. And can we specify the type?

ex:

private List<String> ...

Reply:
Done.